### PR TITLE
Error while switching tab

### LINKED ISSUE #126: Error while switching tab
Scenario 1:

1) Open a link (https://promactoauth.azurewebsites.net)
2) Login with following credentials:

Email: hardik@promactinfo.com
Password: User00_FHGFDACB

3) Then you may see the detail page of the respective user.
4) Then click on project tab you will get an error.

Please refer the screenshot:

Scenario 2:

1) Open a link (https://promactoauth.azurewebsites.net)
2) Login with following credentials:

Email: hardik@promactinfo.com
Password: User00_FHGFDACB

3) Then you may see the detail page of the respective user.
4) then click on User tab you may see the error.

Please refer the screenshot:

Please try to solve these issues first as its roadblock for me as I am not able to move on with further testing.

### PR DESCRIPTION
**Pull request by iamronakshah**
Fixed #126 

Files Changed:

ProjectRepository.cs - Updated GetAllProjectForUserAsync method.
user-details.html - Added condition when user are admin then show back button 
user-edit.component.ts - updated route path.

### CODE REVIEW

File: Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/ProjectsRepository/ProjectRepository.cs [hunk 1]
```diff
@@ -253,9 +253,9 @@ public class ProjectRepository : IProjectRepository
         /// <returns></returns>
         public async Task<IEnumerable<ProjectAc>> GetAllProjectForUserAsync(string userId)
         {
-            var projects =await _projectDataRepository.FetchAsync(x => x.TeamLeaderId == userId);
+            var projects = _projectDataRepository.Fetch(x => x.TeamLeaderId == userId).ToList();
```
Comment (chintans) [inline Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/ProjectsRepository/ProjectRepository.cs]: why did you removed async calls?

File: Promact.Oauth.Server/src/Promact.Oauth.Server/wwwroot/app/users/user-details/user-details.html [hunk 2]
```diff
@@ -4,7 +4,7 @@
             <h3>User Details</h3>
         </div>
         <div class="text-right items">
-            <button md-raised-button type="submit" class="btn-back btn-green" (click)="goBack()"> <i class="material-icons">arrow_back</i>Back</button>
+            <button md-raised-button type="submit" class="btn-back btn-green" (click)="goBack()" *ngIf="admin==true"> <i class="material-icons">arrow_back</i>Back</button>
```
Comment (chintans) [inline Promact.Oauth.Server/src/Promact.Oauth.Server/wwwroot/app/users/user-details/user-details.html]: there should be === and not ==

File: Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/ProjectsRepository/ProjectRepository.cs (hunk 1 above)
Comment (iamronakshah) [inline Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/ProjectsRepository/ProjectRepository.cs]: sorry sir it is the rebase issue. I have updated it.

File: Promact.Oauth.Server/src/Promact.Oauth.Server/wwwroot/app/users/user-details/user-details.html (hunk 2 above)
Comment (iamronakshah) [inline Promact.Oauth.Server/src/Promact.Oauth.Server/wwwroot/app/users/user-details/user-details.html]: I have update it. now I have use === instead of ==

**Pull request merged**